modules/apps/document-library/document-library-repository-external-api/src/main/java/com/liferay/document/library/repository/external/ExtRepositoryAdapter.java

```diff
@@ -96,8 +96,8 @@ public FileEntry addFileEntry(
 
 		ExtRepositoryFileEntry extRepositoryFileEntry =
 			_extRepository.addExtRepositoryFileEntry(
-				extRepositoryFolderKey, mimeType, fileName, description,
-				changeLog, inputStream);
+				extRepositoryFolderKey, sourceFileName, mimeType, fileName,
+				description, changeLog, inputStream);
```

Comment:
Shouldn't we rename `sourceFileName` -> `fileName` and `fileName` to `title` here for consistency with other methods? We could even get rid of the `fileName` variable and simply update `title` if it is empty.

Reply:
Also, I'd modify the PR description to state that the fileName  comes before the mimeType for consistency with other DL services. Otherwise I fear Brian will this this sorting is arbitrary.

Reply:
thanks
New commit with the rename and added some explanation on the description.